src/components/Modals/FiatRamps/hooks/useFiatRampCurrencyList.ts

```diff
@@ -34,17 +34,13 @@ export const useFiatRampCurrencyList = (fiatRampProvider: FiatRamp) => {
           .map(asset => {
             const reduxAsset = reduxAssets[asset.assetId]
             const fiatBalance = bnOrZero(balances?.[asset.assetId]?.fiat)
-            const minimumSellThreshold = bnOrZero(
```

Comment:
Can we keep this one in the abstraction?

Reply:
as discussed in Discord,
It was comparing the whole balance of an asset before, with current changes, it'll compare it to the selected account balance.